x/auth/legacy/legacytx/stdtx_builder.go

```diff
@@ -34,14 +36,12 @@ func (s *StdTxBuilder) SetMsgs(msgs ...sdk.Msg) error {
 // SetSignatures implements TxBuilder.SetSignatures.
 func (s *StdTxBuilder) SetSignatures(signatures ...signing.SignatureV2) error {
 	sigs := make([]StdSignature, len(signatures))
-
+	var err error
 	for i, sig := range signatures {
-		stdSig, err := SignatureV2ToStdSignature(s.cdc, sig)
+		sigs[i], err = SignatureV2ToStdSignature(s.cdc, sig) // sig.ToStdSignature(cdc)
```

Comment:
Is `ToStdSignature` an idea, or implemented already?

Reply:
An idea. I implemented it, but it created an import cycle, so I rolled it back. I will remove the comment (forgot about it). Thanks.